sys/net/network_layer/ng_ipv6/ng_ipv6.c

```diff
@@ -208,6 +208,7 @@ static int _fill_ipv6_hdr(kernel_pid_t iface, ng_pktsnip_t *ipv6,
 
     DEBUG("ipv6: calculate checksum for upper header.\n");
 
+#if NG_NETIF_NUMOF > 1
     if (payload->users > 1) {
```

Comment:
I don't quite understand how this is related to multiple interfaces. Does this not only check for users (threads currently accessing) of the IP payload?